datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -1001,7 +1018,7 @@ private boolean isRazSupported(String runtime, CloudPlatform cloudPlatform) {
             return true;
         }
         Comparator<Versioned> versionComparator = new VersionComparator();
-        return versionComparator.compare(() -> runtime, () -> AWS.equals(cloudPlatform) ? "7.2.2" : "7.2.1") > -1;
+        return versionComparator.compare(() -> runtime, MIN_RUNTIME_VERSION_FOR_RAZ.get(cloudPlatform)) > -1;
```

Comment:
MIN_RUNTIME_VERSION_FOR_RAZ does not match this condition

Reply:
yes. since we have updated min runtime to 7.2.2 for AWS as part of this change. 7.2.1 is very old and it should not be used.

Reply:
Unfortunately this information was not mentioned neither here in the PR description nor in the Jira ticket.

Reply:
Agreed. It came as part of discussion after Jira was created. I have updated Jira to capture this change as well.